run/image-processing/image.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START run_imageproc_handler_setup]
+const gm = require('gm').subClass({imageMagick: true});
+const fs = require('fs');
+const {promisify} = require('util');
+const path = require('path');
+const {Storage} = require('@google-cloud/storage');
+const storage = new Storage();
```

Comment:
I'd list all `requires` before other variables.

Reply:
Per my PR notes, the image.js file is literally functions/imagemagick/index.js, copied over and with region tags replaced. My goal is not to fork it, hence my previous proposal of uniting them around a code templating tool. Can we move this feedback to a separate issue to the GCF code?

(And of course, feedback welcome on how to approach this idea of shared code across samples)